frontend/src/modules/resourceprogress/components/ResourceProgress.js

```diff
@@ -52,6 +52,10 @@ export class ResourceProgressBase extends React.Component<Props, State> {
         if (total) {
             percent = Math.floor(complete / total * 100);
         }
+        // Do not show resource progress until stats are available
+        else {
+            return false;
```

Comment:
You should return `null` here instead of false, that's the usual when you want to not render a component.